tapdb/supply_commit.go

```diff
@@ -326,6 +342,76 @@ func (s *SupplyCommitMachine) UnspentPrecommits(ctx context.Context,
 			preCommits = append(preCommits, preCommit)
 		}
 
+		// If any pre-commits were found where we acted as the issuer,
```

Comment:
Perhaps this should just be another method? So one used for remote issuers, and one used for local issuers.